### PR TITLE
JBPM-6620: Stunner - Add BPMN Intermediate Catch Message Event

### PR DESCRIPTION
**Pull request by alepintus**
Hi @romartin @hasys @wmedvede ,
Intermediate Catch Message Event added.

Could you take a look please?

Thanks!

### CODE REVIEW

**Comment by romartin** [comment]
ok to test

**Comment by romartin** [comment]
Jenkins execute full downstream build

**Comment by romartin** [comment]
@wmedvede missing your review here plz. Thanks!

**Comment by romartin** [comment]
@wmedvede missing your review here plz. Thanks!

File: kie-wb-common-stunner/kie-wb-common-stunner-sets/kie-wb-common-stunner-bpmn/kie-wb-common-stunner-bpmn-api/src/main/java/org/kie/workbench/common/stunner/bpmn/definition/BaseCatchingIntermediateEvent.java [hunk 1]
```diff
@@ -40,7 +40,8 @@
 
     @Category
     public static final transient String category = Categories.EVENTS;
-
+    @Labels
+    protected final Set<String> labels = new HashSet<String>();
```
Comment (wmedvede) [inline kie-wb-common-stunner/kie-wb-common-stunner-sets/kie-wb-common-stunner-bpmn/kie-wb-common-stunner-bpmn-api/src/main/java/org/kie/workbench/common/stunner/bpmn/definition/BaseCatchingIntermediateEvent.java]: don't know if it's just a matter of how github is presenting the code, but seems like a line break is missing here

**Comment by alepintus** [comment]
Hi @wmedvede,
code changed as suggested.

Thanks!

**Comment by romartin** [comment]
Jenkins execute full downstream build

**Pull request merged**